lib/exprotobuf/decoder.ex

```diff
@@ -1,13 +1,19 @@
 defmodule Protobuf.Decoder do
   use Bitwise, only_operators: true
   alias Protobuf.Field
+  alias Protobuf.OneofField
   alias Protobuf.Utils
 
   # Decode with record/module
   def decode(bytes, module) do
     defs = for {{type, mod}, fields} <- module.defs, into: [] do
       case type do
-        :msg  -> {{:msg, mod}, Enum.map(fields, fn field -> field |> Utils.convert_to_record(Field) end)}
+        :msg  -> {{:msg, mod}, Enum.map(fields, fn field ->
+          case field do
+            %Field{} -> field |> Utils.convert_to_record(Field)
```

Comment:
Style change: could we change these single-piped args to the non-piped version? I've been trying to fix this in my projects because I see it as less than ideal style in a lot of cases.